hooks/submit_for_review.py

```diff
@@ -0,0 +1,322 @@
+        interest to this plugin. Only items matching the filters defined via the
+        item_filters property will be presented to this method.
+
+        A publish task will be generated for each item accepted here. Returns a
+        dictionary with the following booleans:
+
+            - accepted: Indicates if the plugin is interested in this value at
+                all. Required.
+            - enabled: If True, the plugin will be enabled in the UI, otherwise
+                it will be disabled. Optional, True by default.
+            - visible: If True, the plugin will be visible in the UI, otherwise
+                it will be hidden. Optional, True by default.
+            - checked: If True, the plugin will be checked in the UI, otherwise
+                it will be unchecked. Optional, True by default.
+
+        :param settings: Dictionary of Settings. The keys are strings, matching
+            the keys returned in the settings property. The values are `Setting`
+            instances.
+        :param item: Item to process
+
+        :returns: dictionary with boolean keys accepted, required and enabled
+        """
+
+        path = item.properties["path"]
+
+        # log the accepted file and display a button to reveal it in the fs
+        self.logger.info(
+            "Submit for review plugin accepted: %s" % (path,),
+            extra={
+                "action_show_folder": {
+                    "path": path
+                }
+            }
+        )
+
+        # return the accepted info
+        return {"accepted": True}
+
+    def validate(self, settings, item):
+        """
+        Validates the given item to check that it is ok to publish. Returns a
+        boolean to indicate validity.
+
+        :param settings: Dictionary of Settings. The keys are strings, matching
+            the keys returned in the settings property. The values are `Setting`
+            instances.
+        :param item: Item to process
+        :returns: True if item is valid, False otherwise.
+        """
+
+        return True
+
+    def publish(self, settings, item):
+        """
+        Executes the publish logic for the given item and settings.
+
+        :param settings: Dictionary of Settings. The keys are strings, matching
+            the keys returned in the settings property. The values are `Setting`
+            instances.
+        :param item: Item to process
+        """
+
+        sg_publish = item.properties["sg_publish_data"]
+        sg_task = self.parent.context.task
+        comment = item.description
+        thumbnail_path = item.get_thumbnail_as_path()
+        progress_cb = lambda *args, **kwargs: None
+        review_submission_app = self.parent.engine.apps.get("tk-multi-reviewsubmission")
+
+        render_path = item.properties.get("path")
+        render_template = item.properties.get("work_template")
+        publish_template = item.properties.get("publish_template")
```

Comment:
We should ensure these templates aren't `None` and raise appropriately if they are.